### ISSUE TITLE
Synctex not working properly with lualatex

### ISSUE THREAD

**Issue by zlm4** [texstudio-org/texstudio#590]
### Environment
- TeXstudio: git 2.12.14
- Qt: 5.8.0
- OS: Windows 10
- TeX distribution: texlive 2018

### Expected behavior
Compiling the following code with lualatex works. When using the "Go to Source" command from the internal PDF viewer, the corresponding line in the source Code is highlighted.

```\documentclass{book}
\begin{document}
	1
	
	2
	
	3
	
	4
	
	5
	
	6
	
	7
	
	8
	
	9
	
	10
\end{document}
```
### Actual behavior
Above code compiles just fine with lualatex, but "Go to Source" and "Go to PDF" don't work properly. For example, if clicked on the "1" in the PDF, it highlights the "3" in the source code. Above example works fine if compiled with pdflatex or xelatex. It also worked with lualatex with texstudio 2.12.6.

### How to reproduce
Compile above code with lualatex.

**Comment by timhoffm** [comment]
Works on current master. You may try the latest development snapshot: https://texstudio.org/#snapshots

**Comment by zlm4** [comment]
Still doesn't work for me on build 201905040005. It may be important, that the same thing happens with TeXworks. Synctex works properly if I use Sumatra as PDF viewer, so the problem lies probably within the internal PDF viewer.

**Comment by manudroid19** [comment]
> 
> 
> Still doesn't work for me on build 201905040005. It may be important, that the same thing happens with TeXworks. Synctex works properly if I use Sumatra as PDF viewer, so the problem lies probably within the internal PDF viewer.

I have just tested your problem. My setup is MikTeX, [this build](https://github.com/texstudio-org/texstudio/releases/tag/2.12.16beta3) from github betas and windows 10. I added to your code \documentclass{article} before begin document and synctex works fine. Try that beta build, maybe is fixed there. Or maybe is a texlive bug and works fine with miktex.
